pages/home/HomePage.jsx

```diff
@@ -59,7 +60,7 @@ var HomePage = React.createClass({
     return (
       <div>
         <HeroUnit>
-          <h1>The Mozilla Learning Network</h1>
+          <h1>{this.context.intl.formatMessage({id: 'mln'})}</h1>
```

Comment:
I'd advocate clear, human-readable ids. As an acronym can we use "MLN"? (case matters in this instance)